src/panels/config/script/ha-script-editor.ts

```diff
@@ -863,6 +901,9 @@ export class HaScriptEditor extends KeyboardShortcutMixin(LitElement) {
         ha-fab.dirty {
           bottom: 0;
         }
+        ha-fab.saving {
```

Comment:
Doesnt the `disabled` property give it disabled styling? If not that should be fixed in `ha-fab`

Reply:
Will fix this in the `ha-fab`. The `disabled` does not give proper styling